src/analysis/locally_convex/bounded.lean

```diff
@@ -192,6 +192,53 @@ end
 
 end uniform_add_group
 
+section continuous_linear_map
+
+variables [nontrivially_normed_field 𝕜]
+variables [add_comm_group E] [module 𝕜 E]
+variables [uniform_space E] [uniform_add_group E] [has_continuous_smul 𝕜 E]
+variables [add_comm_group F] [module 𝕜 F]
+variables [uniform_space F] [uniform_add_group F]
+
+/-- Construct a continuous linear map from a linear map `f : E →ₗ[𝕜] F` and the existence of a
+neighborhood of zero that gets mapped into a bounded set in `F`. -/
+def linear_map.of_exists_bounded_image (f : E →ₗ[𝕜] F)
```

Comment:
```suggestion
def linear_map.clm_of_exists_bounded_image (f : E →ₗ[𝕜] F)
```
I feel like the name should specify the type of object being constructed here. (Idem for the associated lemmas below.)